schemas/devcontainer-features.schema.json

```diff
@@ -139,6 +139,10 @@
                             "description": "Machine specific settings that should be copied into the container."
                         }
                     }
+                },
+                "deprecated": {
```

Comment:
Probably this file should be deleted ; if it's in use anywhere then should be replaced with 👇  
https://github.com/devcontainers/spec/blob/main/schemas/devContainerFeature.schema.json

Reply:
@joshspicer Do we still need this file here? I think this might just be your first version of it and I copied it over when doing the CLI.

Reply:
correct, we don't need this older file (`devcontainer-features.json` is not a thing anymore).

Reply:
`devcontainer-feature.json` is the file that I just created that new schema definition file for.

Reply:
Good to know. Removed 👍